CPAC/resources/configs/1.7-1.8-deprecations.yml

```diff
@@ -8,7 +8,7 @@
 - targetAngleDeg
 - runSymbolicLinks
 - configFileTwomm
-- ref_mask_2mm
+- ref_mask_res-2
```

Comment:
Not a huge deal, but this will prevent pre-v1.8.0 configs that include a `ref_mask_2mm` key from being updated. I don't know if it matters much though, or how long we're going to continue supporting older pipeline configs

Reply:
We need these templates :)

Reply:
Thanks Xinhui!

And I [reverted that deprecation key](https://github.com/FCP-INDI/C-PAC/commit/14acd06576c326c9790b3336ced4a733d6c98b74)